src/main/groovy/org/shipkit/internal/notes/improvements/GitHubTicketFetcher.java

```diff
@@ -151,7 +153,7 @@ public GitHubIssuesBuilder labels(String labels) {
             GitHubIssues browse() {
                 // see API doc: https://developer.github.com/v3/issues/
                 String nextPageUrl = String.format("%s%s%s%s%s%s%s",
-                        "https://api.github.com/repos/" + repository + "/issues",
+                        apiUrl + "/repos/" + repository + "/issues",
```

Comment:
Replaced `String.format` with simple builder. It resembles URL nature a little bit better. 
I didn't want to rewrite the logic from scratch here, it might be better to do this in subsequent pull request.